source/index.ts

```diff
@@ -260,7 +260,7 @@ is.primitive = (value: unknown): value is Primitive => is.null_(value) || isPrim
 is.integer = (value: unknown): value is number => Number.isInteger(value as number);
 is.safeInteger = (value: unknown): value is number => Number.isSafeInteger(value as number);
 
-is.plainObject = <Value = unknown>(value: unknown): value is Record<string | number | symbol, Value> => {
+is.plainObject = <Value = unknown, Key extends keyof any = string | number | symbol>(value: unknown): value is Record<Key, Value> => {
```

Comment:
1. These changes should be tested, to see the the typing (default vs non-default) work as expected
2. The `assert` method should change as well

Reply:
just changed and pushed